src/store-service.js

```diff
@@ -104,6 +107,16 @@
                 .map((store) => store.id);
         }
 
+        getPreviousClosedWindowNames() {
```

Comment:
I think this function is only called in here? If so it can be performed inline and taken off the interface.